lib/src/geojson.dart

```diff
@@ -276,12 +283,12 @@ abstract class GeometryType<T> extends Geometry {
 
 /// Point, as specified here https://tools.ietf.org/html/rfc7946#section-3.1.2
 @JsonSerializable(explicitToJson: true)
-class Point extends GeometryType<Position> {
-  Point({this.bbox, Position coordinates})
+class Point extends GeometryType<Position?> {
+  Point({this.bbox, Position? coordinates})
```

Comment:
e.g. there is no language in the rfc that suggests a point may have a null coordinates value

Reply:
look at the other comment